src/components/Authorship.svelte

```diff
@@ -35,9 +36,15 @@
   {/if}
   <span class="id layout-desktop">
     {formatNodeId(authorId)}
+    {@html authorAlias
+      ? '<span class="alias">(' + authorAlias + ")</span>"
+      : ""}
   </span>
   <span class="id layout-mobile">
     {formatNodeId(authorId).replace("did:key:", "")}
+    {@html authorAlias
+      ? '<span class="alias">(' + authorAlias + ")</span>"
```

Comment:
when I add a styling for `"alias"` class it doesn't render, e.g.:
```
<style>
...
.alias {
  color: red !important;
}
</style>
```

any idea why?

Reply:
Not sure, but could be because you're rendering with `@html`.

Reply:
yes, works now without `@html`.